### PR TITLE
OP-1130 improve JPA methods- admtype package

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-1130

Changes for the admtype package.

Matching changes for CORE PR: https://github.com/informatici/openhospital-core/pull/1105

### CODE REVIEW

**Comment by dbmalkovsky** [comment]
Closing this PR to try a different approach.

**Comment by dbmalkovsky** [comment]
This method is the one of choice; reopening.

File: bundle/language_en.properties [hunk 1]
```diff
@@ -172,6 +172,7 @@ angal.admission.withthisoperationthepatientwillbedeletedandhisherhistorytransfer
 angal.admission.youareeditinganoldadmission.msg                                                        = You are editing an old admission.
 angal.admisssion.discharedpatientfor.fmt.msg                                                           = Discharged patient: {0} for {1}.
 angal.admtype.admissiontypebrowser.title                                                               = Admission Type Browser
+angal.admtype.admissiontypenotfound.fmt.msg                                                                    = Admission type "{0}" not found.
```
Comment (mwithi) [inline bundle/language_en.properties:175]: alignemnt

**Review by mwithi — APPROVED** [review]
Apart one comment, LGTM. Thanks a lot!

File: bundle/language_en.properties (hunk 1 above)
Comment (dbmalkovsky) [reply bundle/language_en.properties]: It has been corrected; resolving.

**Pull request merged**